itest/lnd_route_blinding.go

```diff
@@ -397,6 +401,108 @@ func setupFourHopNetwork(ht *lntest.HarnessTest,
 	}
 }
 
+// createBlindedRoute creates a blinded route to the recipient node provided.
+// The set of hops is expected to start at the introduction node and end at
+// the recipient.
+func (b *blindedForwardTest) createBlindedRoute(hops []*forwardingEdge,
+	dest *btcec.PublicKey, finalCLTV uint16) *routing.BlindedPayment {
+
+	// Create a path with space for each of our hops + the destination
+	// node. We include our passed final cltv delta here because blinded
+	// paths include the delta in the blinded portion (not the invoice).
+	blindedPayment := &routing.BlindedPayment{
+		CltvExpiryDelta: finalCLTV,
+	}
+
+	pathLength := len(hops) + 1
+	blindedPath := make([]*sphinx.HopInfo, pathLength)
+
+	for i := 0; i < len(hops); i++ {
```

Comment:
Yeah - was lazy here. Updated this to do proper aggregation but went for long-but-readable impl rather than doing something smart (forwards to create payloads / backwards to calc fees) because it's a test function / should be replaced when we start building our own paths.